### ISSUE TITLE
Compatibility

### ISSUE THREAD

**Issue by mstrop** [nullpunkt/lz-string-php#5]
It looks like this PHP class is not compatible with current JS implementation. I have a text:

2	SM-T320	Android	a0c3384d166c62e4	775	1280

JS compresses it (64base) like this:

EwSAygsgtAKgzMADCAggOwCYCcD2BLDEAQ0QGM44AOAFgwEYA2B0h4AU2pAHYuBWEOsEqIgA

This PHP class output is:

EwAjIZQWQWgFQMzAAzhAQQHYBMBOB7AS2zQENkBjBBADgBZsBGANmYueAFM60B2XgKxpGwGsiAA=

**Comment by CityofStw** [comment]
I'm having similar issues, but different.

If I run your string through my versions (.php is current from here, .js is version 1.4.4) I get the same result from each, yet different than either of yours.

EwAgygsgtAKgzMADCAggOwCYCcD2BLDEAQ0QGM44AOAFgwEYA2B0h4AU2pAHYuBWEOsEqIgA

Almost the same as your JS, could just be how I presented the string, dunno.

My issue is that only certain strings will be different.  The sample I am working with currently, if I compressToBase64 on PHP, I can decompress it in PHP or JS.  If I compress the same string in JS, while I can decompress it in JS, it fails in PHP with a "Too much errors" exception.  Only difference is the compressed output is the last three characters:

PHP:  (it is a long string, so snipped the middle out)
N4IgJgpgbglg<snip>YgO8ABdGsQAA=

JS: 
N4IgJgpgbglg<snip>YgO8ABdGsQA==

This isn't consistent, usually it works as expected, but occasionally this will occur.

I have tried the 1.5 beta of the .js, no difference.

Thought I would put this here, before delving into the code, see if anyone had any ideas.
